scripts/prepare_chunks.sh

```diff
@@ -88,4 +99,6 @@ done
 
 # Print BED line
 cat $OUTDIR/regions.tsv | cut -f1-3 | tr -d "\n"
+cat $OUTDIR/nodeColors.tsv
+
```

Comment:
I don't think we want these inserted here; we won't get the right BED line out on standard output.

Also the file might not exist.

```suggestion
```